### DISCUSSION TITLE
自动导入标注到 Obsidian（不使用同步助手方案）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3932] [服务]
> 导入到 Obsdian 插件升级到 1.2.0，增加了自动导入标注到 Obsidian 的功能，此文为一站式配置教程，查看此插件的文档说明，请 [前往这里](https://github.com/Kenshin/simpread/discussions/2902)。

## 配置库

[简悦 · 配置库](https://github.com/Kenshin/simpread/discussions/4531) 是简悦官方推出的一套针对新用户的极简配置方案，方便新用户用最快的方式使用简悦的各种高级服务，配置库内置了常用的双链笔记用法，如：Notion、Obsidian、Logseq、Roam Research，同时包含了简悦在阅读模式上的一些常规插件：Live Editor、题图、Safari 阅读模式等。

目前配置库包含了 Obsidian 的一站式配置方案，细节 [请看这里](https://github.com/Kenshin/simpread/discussions/4531#discussioncomment-3844881)。

## 视频

https://user-images.githubusercontent.com/81074/164135947-0552fc11-1b92-4804-a616-df64ae43770a.mp4

## 特点

不使用 [同步助手](http://ksria.com/simpread/docs/#/Sync?id=导出服务) 方案，即可将标注导入到 Obsidian 指定位置，适合轻量级用户使用。

## 监控

1. 稍后读的加入、稍后读元数据的改变
2. 标注的加入、删除。
3. 标注色、标注元数据的改变。

## 前提

1. 此插件的版本为 1.2.0 版，如果你已经安装过这个插件，请需要升级，如何查看和升级请看下面。
2. 简悦需升级到 [高级账户](http://simpread.pro/price)。
3. 事先安装好 Obsidian Local REST API 插件。
4. 需要配置简悦的 [知识库](https://kb.simpread.pro/#/page/建立知识库)。（根据教程即可）

![img](https://pic2.zhimg.com/80/v2-3546d2ac9fb6eadeaedfa19cdac78e74_1440w.jpg?source=d16d100b)

### 下载

https://simpread.ksria.cn/plugins/details/1VQ19jCD8Z

### 设置 Obsidian 库文件名

![img](https://pic2.zhimg.com/80/v2-6ae713a45250a3987aecdd198918bd4d_1440w.jpg?source=d16d100b)

### 获取文件夹名称

文件夹建议事先建好

![img](https://pic2.zhimg.com/80/v2-b13c2ca12464f12906666c607991b0e9_1440w.jpg?source=d16d100b)

### 设置库名称与文件夹名称

![img](https://pic1.zhimg.com/80/v2-a0a10ec8591f85d04b0101fd83c62f03_1440w.jpg?source=d16d100b)

### 获取 Local REST API

![img](https://pic3.zhimg.com/80/v2-b3b7e23f6bd3d5716fc4ef1c24dd0b85_1440w.jpg?source=d16d100b)

### 设置 Token

![img](https://pic2.zhimg.com/80/v2-b4ff4b9e253bf3b5f2133c938702c2e2_1440w.jpg?source=d16d100b)

### 开启 Local REST API 导出方案

![img](https://pic3.zhimg.com/80/v2-9fda71dd55304bca5083404f2de2f5c5_1440w.jpg?source=d16d100b)

### 设置文件名

不要更改下面的内容

```
{{id}}-{{title}}{{mode}}
```

![img](https://pic2.zhimg.com/80/v2-e1f85d877584c34996330efa3231b241_1440w.jpg?source=d16d100b)

### 自动化设置

开启下面的选项

![img](https://pic3.zhimg.com/80/v2-c909dda81c6cbafc36364a3987fbe9fb_1440w.jpg?source=d16d100b)

### 保存

至此导入到 Obsidian 选项设置完毕，保存后刷新页面看下是否生效。（如不生效请看此页面下面的解决方案）

![img](https://pic1.zhimg.com/80/v2-f658dd7de195a76da636cab073717756_1440w.jpg?source=d16d100b)

### 下载插件

https://simpread.ksria.cn/plugins/details/DH9l5jblPH

### 配置

将下面的代码

```
o b u n
o b u u
o b a a
o b a c
```

加入到下图即可。

![img](https://pica.zhimg.com/80/v2-12e84df3a1db15d6e341ae1ffb5ad96a_1440w.jpg?source=d16d100b)

### 保存

至此 [自动化辅助增强插件](https://github.com/Kenshin/simpread/discussions/3596) 完毕，保存后刷新页面看下是否生效。（如不生效请看此页面下面的解决方案）

![img](https://pica.zhimg.com/80/v2-942ea3574f9661dcf1f6b97682300cd3_1440w.jpg?source=d16d100b)

## 扩展端设置

扩展端仅需要设置下面的选项，位置在：选项页 → 服务 → 定制导出，同样此功能需要高级账户支持。

### 开启自定义标题

![img](https://pic3.zhimg.com/80/v2-dc66f13d9719a1a7c3b1bdc9b2cb678f_1440w.jpg?source=d16d100b)

## 自定义 Markdown 模板（旧）

通过 [简悦官方模板库](https://github.com/Kenshin/simpread/discussions/2153) 随便找到一个 Obsidian 模板复制到选项页 → 服务 → 定制导出

![img](https://pica.zhimg.com/80/v2-80ef63f0c2393955306551b9601b7a6e_1440w.jpg?source=d16d100b)

注意：不建议使用此方式的自定义 Markdown 功能，请继续看新方案。（此方案依旧可以使用，只是没有新方案的模板更灵活），使用此功能时，需要关闭下面的选项。

![img](https://pic1.zhimg.com/80/v2-e973b9fafd363c3965b208ce0928c08d_1440w.png?source=d16d100b)

## Markdown 辅助增强方案（新）

这是简悦新推出的一个插件，用来弥补扩展端 Markdown 语法规则一些弊端而产生的，细节 [请看这里](https://github.com/Kenshin/simpread/discussions/3725)。

### 如何使用

需开启下面的选项。

![img](https://pic2.zhimg.com/80/v2-36f8fc069b92c333de0f63d7e5490173_1440w.png?source=d16d100b)

随便进入一个页面 e.g. https://sspai.com/post/71576 并进入 **阅读模式 → 右下角触发器 → 动作 → 插件触发器**

在 [此模板库](https://github.com/Kenshin/simpread/discussions/3725#discussioncomment-2635600) 随便找一个对应的模板，并按照下图所示即可查看此插件的选项页。（注意：如果你的界面没有插件触发器，请根据此 [解决方案](https://github.com/Kenshin/simpread/discussions/2342) 修复。

![img](https://pic1.zhimg.com/80/v2-6c7824fbdaa439c64f4d8d547c78b286_1440w.png?source=d16d100b)

## 最后

至此全部配置已经设置完毕，这些只是基础设置，此插件支持更多设置项，请前往 [文档查看](https://github.com/Kenshin/simpread/issues/2900)。

### 与同步助手方案的区别

1. 同步助手方案是通过自动化将标注导入到 output 文件夹，支持阅读模式与稍后读使用，[配合 Hazel 自动化工具](https://github.com/Kenshin/simpread/discussions/3241)可不用开着 Obsidian。
2. 此插件不使用同步助手，但需要安装  Obsidian Local REST API 插件以及 [自动化辅助增强插件](https://simpread.ksria.cn/plugins/details/DH9l5jblPH)，只能在阅读模式下使用，无法在稍后读下使用，适合不希望配置以及轻量级用户使用，无需 Hazel 等自动化工具但需要开着 Obsidian 才能导入成功。

### 插件安装失败等原因的解决方案

如果你没有使用同步助手的话，一般不会出现此问题，但发生无法安装插件等问题，请重置插件解决。

![img](https://pica.zhimg.com/80/v2-ff4f678d9bdd0788bf32eb2953957780_1440w.jpg?source=d16d100b)

如果你使用了同步助手的话，请查看 [此解决方案](https://github.com/Kenshin/simpread/discussions/2342)，更多相关问题，请查看此 [问题清单](https://github.com/Kenshin/simpread/discussions/categories/重要通知?discussions_q=category%3A重要通知+插件)。

### 如何升级插件

选项页 → 插件管理 → 导入到 Obsidain → 第三个 icon（或者直接 [重新安装](https://simpread.ksria.cn/plugins/details/1VQ19jCD8Z)）。

![img](https://pic3.zhimg.com/80/v2-794e2bad40a33d86018da83a7dde0cf2_1440w.jpg?source=d16d100b)

**Comment by YB-Kang** [comment]


**Reply by Kenshin** [reply]
看错误提示是：因为空格导致文件失效。

是否安装了 https://simpread.ksria.cn/plugins/details/tMGXrU1v0U 另外，可以找个没有空格的页面测试下 e.g.  https://sspai.com/post/69972

**Reply by YB-Kang** [reply]
我这两天尝试清空本地插件、重装简悦插件等等都不行，刚才把Obsidian端的简悦同步插件先禁用了，结果可以用了，再重新打开Obsidian的同步插件也没问题了。。

另外，说到OB端的同步插件，我前几天更新了2.0版，发现导入的每条标注都包含html标签，在1.0版本是可以把html标签转换掉的。

![image](https://user-images.githubusercontent.com/55050981/169689406-4b8cd9bb-b55c-4c7c-83bb-15abb4e1e2e0.png)

**Reply by YB-Kang** [reply]
[Obsidian 端的简悦同步插件](https://github.com/Kenshin/simpread/discussions/2889)

**Reply by Kenshin** [reply]
这个插件近期会升级，先不要使用旧版本（当前版本）

**Reply by YB-Kang** [reply]
好的 辛苦了

**Reply by KK-3458** [reply]
![image](https://user-images.githubusercontent.com/66560506/175805070-31286f2f-6622-43a3-b969-79f5bf104080.png)
怎么没看到这个选项😿

**Reply by Kenshin** [reply]
@KK-3458

关闭自动同步，再安装，配置完再重新开启自动同步。

**Reply by PenguinDang** [reply]
安装了`Markdown 辅助增强`之后在插件触发器里找不到这个插件的选项

**Reply by Kenshin** [reply]
@PenguinDang 

解决方案 https://www.yuque.com/kenshin/simpread/fcw60u

**Comment by Kenshin** [comment]
嗨，各位。

简悦官方针对新用户或不喜欢配置的用户，发布了一个新功能：**Obsidan 配置库**。

细节请看下面，几分钟就能配置成完成。

***

![img](https://imgs.zhubai.love/c626678ff6b64dd1ac9dfdae6cf6d188.png)

> 针对新用户的一步到位的 Obsdian 配置库。

## 🗂 配置库

[简悦 · 配置库](https://www.yuque.com/kenshin/simpread/ds8zk0) 是简悦官方推出的一套针对新用户的极简配置方案，方便新用户用最快的方式使用简悦的各种高级服务，配置库内置了常用的双链笔记用法，如：Notion、Obsidian、Logseq、Roam Research，同时包含了简悦在阅读模式上的一些常规插件：Live Editor、题图、Safari 阅读模式等。

## 💡 特点

用户仅需要下载对应的配置库，修改里面的 UID 为自己的高级账户 UID，即可实现简悦的各种高级操作。

## 💁‍♂️ 受众

新用户或者不想使用手动配置的用户。

## 📝 如何使用

1️⃣ 下载

2️⃣ 修改为自己的 UID

3️⃣ 将必要的文件夹复制粘贴到相应位置

✅ Done！

## 📚 Obsidian 配置库

## 📝 特点

1️⃣ 加入稍后读，自动将本地快照和 Markdown 导入到 Obsidian

2️⃣ 加入标注时，自动将标注内容导入本地快照和 Obsidian

3️⃣ 改变稍后读或标注的元数据时，自动将改动的内容保存到本地快照和 Obsidian

4️⃣ 在 Obsidian 中直接内置好了简悦的标注系统，可直接在 Obsidian 实现标注

## 📖 区别

![img](https://imgs.zhubai.love/f4a51d8e8fb4401281b07f82bf0bc7d1.png)

## 📘 极速版

🔗 [Github](https://github.com/Kenshin/simpread-configs/blob/main/obsidian@little/Getting%20Started.md) | [语雀](https://www.yuque.com/kenshin/simpread/xkuecp)

不需要同步助手，无任何配置，支持 Markdown 形式的快照，自动同步标注，适合 Obsidain + 简悦的尝鲜用户。

## 📙 基础版

🔗 [Github](https://github.com/Kenshin/simpread-configs/blob/main/obsidian@localrestapi/Getting%20Started.md) | [语雀](https://www.yuque.com/kenshin/simpread/cg33gh)

需要同步助手，几乎无任何配置，支持本地快照，自动同步标注，适合 Obsidain + 简悦的轻量级用户。

## 📗 高级版

🔗 [Github](https://github.com/Kenshin/simpread-configs/blob/main/obsidian@sync/Getting%20Started.md) | [语雀](https://www.yuque.com/kenshin/simpread/wq35mh)

需要同步助手，几乎无任何配置，支持本地快照，自动同步标注，可在 Obsidian 中标注，适合 Obsidain + 简悦的重度用户。
